### DISCUSSION TITLE
Mettre en place un système de binome dev

### DISCUSSION THREAD

**Discussion by LucasCharrier** [betagouv/cap#12] [Besoin]
### Besoin

```bash
Aider les devs de la commmunauté qui sont seuls à trouver des gens avec qui binomer.
```

### Informations complémentaires

- [X] J'avais déjà entendu ce besoin au sein de la communauté beta.gouv
- [X] Je me porte volontaire pour participer à la mise en place de la future solution choisie

**Comment by damienromito** [comment]
J'ai fait ça pendant 3 mois lorsque je suis arrivé chez beta gouv mais avec un développeur externe. 

Concrétement, tous les jours travaillés, nous nous retrouvions en fin de journée entre 15 et 30 min pour rédiger ensemble la liste des taches réalisés dans la journées et des problèmes rencontrés afin de le partager à mon équipe. (très structurant et rassurant pour l'équipe)
Dans mon cas, ce développeur relisait mon code et me faisais des retours constructifs. 

Je pense que quelque chose peut être adaptés pour les nouveaux arrivants, à coupler peut être avec le système de parrainage.

Bref, je me porte volontaire pour participer à la mise en place de la future solution choisie.

**Comment by chaibax** [comment]
Ca serait top de pouvoir identifier et faire se rencontrer les solo dev de la communauté .

**Comment by revolunet** [comment]
Une piste possible : dans l'espace membre, proposer des pairs randomm en fonction de ton "Domaine" avec un CTA pour rentrer en contact : "découvre tes pairs"

**Comment by MichaelBitard** [comment]
Côté Camino nous travaillons quasiment exclusivement en paire. 

Une idée comme ça pourrait être de pair-programmer de temps en temps avec un autre dev d'une autre startup.

En gros je vais pairer 1/2 journée sur une startup avec un dev, et après ce même dev vient pairer avec moi sur Camino.

Comme ça côté 'consommation Jours' on est à l'équilibre, et on peut discuter/pairer cross-startups.

**Reply by ikarius** [reply]
Je trouve l'idée intéressante, mais ça implique quand même une connaissance approfondie du métier de la startup cible sur les tâches de dev pointues.

**Reply by MichaelBitard** [reply]
Oui je suis assez d'accord avec toi.

Si on parle d'une demi journée de pair-programming, ça implique quand même une tache finissable dans ce temps là (ou un bout de tache a minima un peu cadrée)

Si c'est un truc qui fonctionne et qui est amené à se refaire sur la même startup, la complexité des taches peut augmenter :)

**Comment by revolunet** [comment]
Et si on essayait de craquer ce sujet au prochain forum tech, le 6 février ?

**Reply by MichaelBitard** [reply]
J'aurais du proposer ce sujet au dernier forum en ligne, je ne pourrais pas être avec vous le 6 février en présentiel.

**Comment by damienromito** [comment]
Hello Julien, J’espère que tu vas bien. Sujet très intéressant, j’essaie
toujours d’être dans une dynamique similaire, c’est très enrichissant . Je
ne suis plus à Betagouv depuis septembre, j’ai souhaité remettre mes
compétences au service de l’éducation. J’espère en tout cas que le sujet
pourra continuer à être travaillé, si besoin de retours d’expérience, je
reste dispo.
Bonne journée
Damien
